content/docs/intro/concepts/stack.md

```diff
@@ -61,24 +67,48 @@ test                                      2 weeks ago              121
 To select a stack that is part of an organization, use the fully-qualified stack name, either `orgName/stackName` or `orgName/projectName/stackName`:
 
 ```bash
-$ pulumi stack select acmecorp/prod
+$ pulumi stack select mycompany/prod
 
 $ pulumi stack ls
 NAME                                      LAST UPDATE              RESOURCE COUNT
-acmecorp/prod*                            4 hours ago              97
-acmecorp/staging                          4 hours ago              97
+mycompany/prod*                            4 hours ago              97
+mycompany/staging                          4 hours ago              97
 dev                                       n/a                      n/a
 ```
 
 ## Deploy a project
 
-To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "config" >}}) for the active stack.
+To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "/docs/intro/concepts/config" >}}) for the active stack.
+
+{{% notes "info"%}}
+Your stack can distinguish between execution for `preview` versus for `update` by using [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
```

Comment:
```suggestion
Your program code can distinguish between `preview` and `update` operations by using  [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
```

Worth using a shortcode here to make this link a language-specific one?

Reply:
Implemented the wording changes, but will capture the shortcode suggestion in a follow-up issue.